### ISSUE TITLE
InvokeSimple in job.go always returns false

### ISSUE THREAD

**Issue by git-ktse** [bndr/gojenkins#175]
Line 293 in job.go

	if resp.StatusCode != 200 || resp.StatusCode != 201 {
		Error.Println("Could not invoke job %s", j.GetName())
		return false
	}

Should be:

	if resp.StatusCode != 200 && resp.StatusCode != 201 {
		Error.Printf("Could not invoke job %s: %d", j.GetName(), resp.StatusCode)
		return false
	}

**Comment by betterfor** [comment]
You may in v0.2.0.
Now master is
`if resp.StatusCode != 200 && resp.StatusCode != 201 
{
	return 0, fmt.Errorf("Could not invoke job %q: %s", j.GetName(), resp.Status)
}`

**Comment by git-ktse** [comment]
Yes, I did see the fix in master.  How can I get the latest code?  I deleted the pkg, and did a get again, and still saw the old code.

**Comment by StantonMatt** [comment]
This is fixed on current `master`.

`Job.InvokeSimple` now returns `(int64, error)` with the Jenkins queue item id, and the status-code guard uses the corrected condition:

```go
if resp.StatusCode != 200 && resp.StatusCode != 201 {
    return 0, fmt.Errorf("could not invoke job %q: %s", j.GetName(), resp.Status)
}
```

`Jenkins.BuildJob` delegates through this path, so successful 200/201 responses continue to parse the `Location` queue item URL instead of failing the old boolean branch.

I rechecked current `master` with:

`go test . -run 'TestJenkins_BuildJob_Success|TestJob_InvokeSimple_Success' -count=1 -v`

Both focused tests pass locally.
